app/javascript/components/rules/RuleNavigator.vue

```diff
@@ -124,7 +124,7 @@
       <!-- Currently opened controls -->
       <p class="mt-0 mb-0">
         <strong>Open Controls</strong>
-        <template v-if="openRuleIds.length > 0" href="#">
+        <template v-if="openRuleIds.length > 0">
```

Comment:
what was the purpose of this change?  clicking on something that has href=# makes it go to the top of the page.  is this behavior no longer required?

Reply:
That's still the case for pure html. Template attributes as useless when used with v directives (if, else-if, else, for, slot ).. The behavior here is to close all open controls and currently does not scroll to the top after the click action. Thus useless. This is flagged by the vue/no-useless-template-attributes.